build-packages.mjs

```diff
@@ -31,8 +31,10 @@ const libDir = path.join(packageDir, 'lib');
 const typesDir = path.join(packageDir, 'types');
 
 const [sourceFiles] = await Promise.all([
-  // Find all .js and .ts files from /src
-  glob(`${sourceDir}/**/*.{mjs,cjs,js,mts,cts,ts}`, { onlyFiles: true, absolute: false }),
+  // Find all .js and .ts files from /src.
+  // `fast-glob` doesn't find any files when `sourceDir` contains "forward slashes"
+  // (Windows) instead of "back slashes" (Linux).
+  glob(`${sourceDir.replaceAll('\\', '/')}/**/*.{mjs,cjs,js,mts,cts,ts}`, { onlyFiles: true, absolute: false }),
```

Comment:
Can we do the replaceAll in line 17 where we set the rootDir? And is there instead of fileURLToPath a more OS agnostic function we can alternatively use? Or is replaceAll the easier solution? I don't have any experience with this

Reply:
@WikiRik Hmm, interesting thought.

Looking at the `fileURLToPath()` function, seems like it's a known issue with it and Windows slashes: https://github.com/nodejs/node/issues/25265
According to that issue, originally it returned the universal slashes and then they specifically changed it to windows-style slashes in https://github.com/nodejs/node/commit/7237eaa3353aacf284289c8b59b0a5e0fa5744bb

So I guess, `replaceAll()` is the only solution when using that specific function.